android/src/main/java/cz/ackee/extensions/android/DrawableExtensions.kt

```diff
@@ -7,14 +7,15 @@ import android.graphics.Color
 import android.graphics.drawable.*
 import android.os.Build
 import androidx.annotation.ColorInt
+import androidx.core.graphics.ColorUtils
 
 // Extensions for creating drawables and other drawable-related helpers
 
 fun backgroundDrawable(
     @ColorInt color: Int,
     isButton: Boolean = false,
     @ColorInt checkedColor: Int = color,
-    @ColorInt pressedColor: Int = color.toDarkerColor(),
+    @ColorInt pressedColor: Int = if (ColorUtils.calculateLuminance(color) < 0.5) color.toLighterColor() else color.toDarkerColor(),
```

Comment:
😐 please introduce some extension that will check this with self describing name like `isDarkColor`

Reply:
155570afc56fbb6aaf2b92b38381e7615b9b471f